ci/build.yml

```diff
@@ -198,7 +198,7 @@ jobs:
         # and then splice that in via a template parameter.
         skip_tests: $(skip_tests)
         is_release: variables.is_release
-        is_split_release: variables.is_split_release
+        is_split_release: $(is_split_release)
```

Comment:
YAML programming is hard. I don't know why the `variables.` syntax seems to work for `is_release` and not for `is_split_release`, but maybe this other syntax will work. 🤷